torchvision/prototype/models/detection/keypoint_rcnn.py

```diff
@@ -56,17 +56,17 @@ class KeypointRCNN_ResNet50_FPN_Weights(WeightsEnum):
             "map_kp": 65.0,
         },
     )
-    default = Coco_V1
+    default = COCO_V1
```

Comment:
I think all the `default` aliases in all enums need to become `DEFAULT` as well. Thoughts?

If we do this, there are a couple of places the default is referenced as a string in the code that we also need to change. Example [1](https://github.com/pytorch/vision/blob/main/torchvision/prototype/models/_utils.py#L81) and [2](https://github.com/pytorch/vision/blob/main/test/test_prototype_models.py#L58).

Reply:
@datumbox Yes, I agree with replace `default` with `DEFAULT`

Reply:
done